README.md

```diff
@@ -16,6 +16,7 @@ Format the values for these config options per the [PHP_CodeSniffer documentatio
 
 1. file_extensions - This is where you can configure the file extensions for the files that you want PHP_CodeSniffer to analyze.
 2. standard - This is the list of standards that you want PHP_CodeSniffer to use while analyzing your files.
+3. ignore_warnings - You can hide warnings, and only report errors with this option.
```

Comment:
I know it's not entirely your change, but could we turn this into an unordered list instead of an ordered list?